src/sagemaker_xgboost_container/data_utils.py

```diff
@@ -60,6 +60,16 @@ def get_content_type(content_type_cfg_val):
         return LIBSVM
     elif content_type_cfg_val.lower() in [CSV, _content_types.CSV]:
         return CSV
+    elif _content_types.CSV in content_type_cfg_val.lower():
+        items = content_type_cfg_val.split(';')
+        label_size = [item for item in items if 'label_size' in item]
+        if label_size:
+            label_size = label_size[0].split('=')[1]
+            if label_size.strip() != '1':
```

Comment:
How about making 1 as a parameter?

Reply:
What would be the benefit in this case? By definition, label size for XGBoost must be 1, and this value is only referenced twice.